app/utils/SystemPropertiesHelper.scala

```diff
@@ -62,13 +62,16 @@ trait SystemPropertiesHelper {
     }
 
   def doesntExist[T](key: String, defaultvalue: T): T = {
-    Logger.info(s"No system property $key defined. Using default value: $defaultvalue")
+    Logger.error(
```

Comment:
How frequently do we get this log? Same question for the doesntParse method. I'd be wary if this ends up flooding our kibana logs.

Reply:
it seems to come in waves more than anything, 2-3 spikes within the last 14 days